src/__tests__/Batcher_test.ts

```
@@ -115,6 +115,94 @@ describe("Batcher", () => {
+      undefined,
+      undefined,
+      { outputReference: "meta-x7k2q" }
+    );
+
+    const events: Array<SessionStartedBody | EventSchemaBody> | null = AvoInspector.avoStorage.getItem(AvoBatcher.cacheKey);
+
+    expect(events).not.toBeNull();
+    if (events !== null) {
+      expect(events.length).toEqual(1);
+      expect((events[0] as EventSchemaBody).outputReference).toEqual("meta-x7k2q");
+      expect(events[0].hasOwnProperty("originHint")).toEqual(false);
```

ESLint

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

(no-prototype-builtins)